database/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:latest
+FROM postgres:16.1
 ENV POSTGRES_PASSWORD=secret
```

Comment:
Note: don't save any passwords in github
create .env file, and create .env.example file(you can push this file), update dockerfile to use .env file instead passwords and credentials